CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -1001,26 +1071,31 @@ public let DECL_NODES: [Node] = [
         name: "Attributes",
         kind: .collection(kind: "AttributeList", collectionElementName: "Attribute"),
         nameForDiagnostics: "attributes",
+        description: "Attributes attached to the import declaration.",
         isOptional: true
       ),
       Child(
         name: "Modifiers",
         kind: .collection(kind: "ModifierList", collectionElementName: "Modifier"),
         nameForDiagnostics: "modifiers",
+        description: "Modifiers attached to the import declaration",
```

Comment:
AFAICT these modifiers aren’t used by any language feature yet. It’s fine that we still parse them but I think it’s worth pointing that out.

```suggestion
        description: "Modifiers attached to the import declaration. Currently, no modifiers are supported by Swift.",
```